lib/email/utils/helpers.js

```diff
@@ -56,7 +54,13 @@ function logEmailEventSent(log, message) {
   emailEventInfo['flow_id'] = getHeaderValue('X-Flow-Id', message)
   emailEventInfo.locale = getHeaderValue('Content-Language', message)
 
-  log.info(emailEventInfo)
+  const addrs = [message.email].concat(message.ccEmails || [])
+
+  addrs.forEach(addr => {
+    const msg = Object.assign({}, emailEventInfo)
```

Comment:
I originally just altered the `emailEventInfo` object in this loop, to try to prevent creating unnecessary objects, but found that didn't play well with `sinon`s spies in our `spyLog`. It just captures the arguments by reference, and so at the end of the test, the 3 arguments were identical.

This probably wouldn't happen to our real logger, since it converts the object to JSON before returning, but if some asynchronous code were ever inserted somewhere in that pipeline, we'd hit the same problem.

Therefore, we clone the object instead. Thanks JS!